### DISCUSSION TITLE
Resume reviews

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#1030] [Tips and Tricks]
Top tips from live stream https://youtu.be/yoqm0K_V_1c

- Single column resume for platforms that digest resume (multi column looks good but might end up as a mess in a platform / service)
- Links to projects / events
- Links in text as well as clickable (just in case it is printed)
- Skills on the top
- Bullet points where it makes sense to have short sentences (not single bullet points)
- Work experience before education
- Lists to have 3-5 items and then have a call to action "more available on request" with a number in brackets
- AlexJS to check for inclusive terminology 
- Remove redundant words by grouping sections together 
- Don't use blue font colour as it can be confused with links
- Pay attention to fonts and white space
- Call to action top/bottom of resume (maybe to visit your GitHub profile or YouTube video)

**Comment by Nitesh-thapliyal** [comment]
- Mention college CGPA/GPA and school percentage only when it is worth showing otherwise it gives a negative impact on the interviewer

**Comment by Rahma-AlWadhahi** [comment]
Links to two services that has template designs for your resume

- [Novo resume](https://novoresume.com/) 
- [CV Designr](https://cvdesignr.com/en) 

shared by Nick and Alex

**Reply by eddiejaoude** [reply]
Thank you for documenting 👍

**Reply by kaiwalyakoparkar** [reply]
OverLeaf is a great latex based resume creator

**Comment by jai-dewani** [comment]
Have skills divided into sections so they only take up one line for each section (for one column resume)

**Reply by Rahma-AlWadhahi** [reply]
Speaking of skills, I found this image which describes the difference between soft skills and hard skills
![image](https://user-images.githubusercontent.com/61970868/109565152-7c8e1b80-7afb-11eb-945a-c55dff1a2008.jpeg)

**Reply by DevCassie** [reply]
> Speaking of skills, I found this image which describes the difference between soft skills and hard skills
> ![image](https://user-images.githubusercontent.com/61970868/109565152-7c8e1b80-7afb-11eb-945a-c55dff1a2008.jpeg)

That's useful! Great find!

I myself refer soft skills as competences. (Not sure if I made the right translation for competences)

**Comment by DevCassie** [comment]
* Don't overuse colour. Keep accent colours to the things that need accent like headings. (Depends on your resume design).
* I don't fully agree with don't use blue font colour. Just don't use the same looking blue font colour on text :p. 
* Keep padding and alignment consistent. 
![image](https://user-images.githubusercontent.com/19227530/109546294-47b5a080-7aca-11eb-8867-05cb7cfc6eac.png)
* For a modern age, use modern (Sans-Serif) fonts.
* Be creative with lists, you could use something or create your own lists with a custom icon and not the standard bullet point.

I don't own the image, this image can be found here all credits go to the rightful owner:
https://uxengineer.com/principles-of-design/white-space/
